docs/src/components/ComponentPlayground.tsx

```diff
@@ -12,6 +12,17 @@ type ComponentPlaygroundProps = {
   componentName: string
 }
 
+const NoopKnobProvider: React.FunctionComponent = props => {
+  const knobContext = React.useContext(unstable_KnobContext)
+  const noopContext = { ...knobContext, registerKnob: _.noop, unregisterKnob: _.noop }
+
+  return (
+    <unstable_KnobContext.Provider value={noopContext}>
```

Comment:
I don't have a better idea now, but I definitely want to remove this from knobs:

![image](https://user-images.githubusercontent.com/14183168/59360006-b3247100-8d2f-11e9-8e53-06a11941f539.png)

Reply:
The Tooltip component was merged, please consider those changes as well.

Reply:
Done 👍